### PR TITLE
CDAP-6279 Fix log metrics test by waiting for metrics aggregation to happen

### PR DESCRIPTION
**Pull request by poornachandra**
JIRA - https://issues.cask.co/browse/CDAP-6279
Build - http://builds.cask.co/browse/CDAP-DUT4257-9

### CODE REVIEW

**Comment by gokulavasan** [comment]
LGTM 👍

**Comment by poornachandra** [comment]
Verified the fix by running the test more than 500 times successfully.

File: cdap-watchdog/src/test/java/co/cask/cdap/logging/save/LogSaverPluginTest.java [hunk 1]
```diff
@@ -128,21 +129,27 @@ public static void initialize() throws IOException {
...
 
-  public void startLogSaver() throws Exception {
     txManager = injector.getInstance(TransactionManager.class);
     txManager.startAndWait();
+    metricsCollectionService.startAndWait();
+  }
+
+  @AfterClass
+  public static void shutdown() {
+    appender.stop();
```
Comment (gsps1) [inline cdap-watchdog/src/test/java/co/cask/cdap/logging/save/LogSaverPluginTest.java:140]: where do we start the appender ?

**Comment by gokulavasan** [comment]
LGTM 👍

File: cdap-watchdog/src/test/java/co/cask/cdap/logging/save/LogSaverPluginTest.java (hunk 1 above)
Comment (poornachandra) [inline cdap-watchdog/src/test/java/co/cask/cdap/logging/save/LogSaverPluginTest.java:140]: It is started when `LogAppenderInitializer` is initialized.

**Comment by gsps1** [comment]
lgtm, squash and merge after build.

File: cdap-watchdog/src/test/java/co/cask/cdap/logging/save/LogSaverPluginTest.java (hunk 1 above)
Comment (gsps1) [inline cdap-watchdog/src/test/java/co/cask/cdap/logging/save/LogSaverPluginTest.java:140]: ok

**Comment by poornachandra** [comment]
Build is green, merging.

**Pull request merged**